e2e/localbitsong/scripts/setup.sh

```
@@ -0,0 +1,146 @@
+BITSONG_HOME=$HOME/.bitsongd
+CONFIG_FOLDER=$BITSONG_HOME/config
+MONIKER=val
+STATE='false'
+
+MNEMONIC="bottom loan skill merry east cradle onion journey palm apology verb edit desert impose absurd oil bubble sweet glove shallow size build burst effort"
+
+while getopts s flag
+do
+    case "${flag}" in
+        s) STATE='true';;
+    esac
```

Shellcheck (0.10.0)

[warning] Invalid flags are not handled. Add a *) case.

(SC2220)